htdocs/langs/en_US/banks.lang

```diff
@@ -148,7 +148,7 @@ BackToAccount=Back to account
 ShowAllAccounts=Show for all accounts
 FutureTransaction=Transaction in futur. No way to conciliate.
 SelectChequeTransactionAndGenerate=Select/filter checks to include into the check deposit receipt and click on "Create".
-InputReceiptNumber=Choose the bank statement related with the conciliation. Use a sortable numeric value (such as, YYYYMM)
+InputReceiptNumber=Choose the bank statement related with the conciliation. Use a sortable numeric value: YYYYMM or YYYYMMDD
```

Comment:
I am afraid I expressed myself badly.

I meant that it would be nice to correct the string so that let the users know that we are talking about alphanumeric order, not about numeric order. YYYYMM or YYYYMMDD are examples but  they are not enforced formats yet.